src/util.ts

```diff
@@ -727,13 +753,41 @@ export class Util {
     config: MakeRequestConfig,
     callback: BodyResponseCallback
   ): void | Abortable {
+    let autoRetryValue = AUTO_RETRY_DEFAULT;
+    if (
+      config.autoRetry !== undefined &&
+      config.retryOptions?.autoRetry !== undefined
```

Comment:
Will `config.retryOptions` always be an object (never undefined)?

Reply:
It will not always be an object. In this test, it is undefined. https://github.com/googleapis/nodejs-common/blob/9bd6d9e16ff0102b526860ecbc2f33985bca4959/test/util.ts#L1420

Reply:
Nice, thanks!